config/install/embed.button.media_browser.yml

```diff
@@ -15,4 +15,4 @@ type_settings:
   entity_browser: media_browser
   entity_browser_settings:
     display_review: false
-icon_uuid: null
+icon_uuid: 3ed72229-7529-4cf5-b679-1d7a86b1030a
```

Comment:
Is this a reference to a file?

Reply:
Ah, I see it is and you have hardwired it in the module. Ok. Ignore this comment.

Reply:
yeah. it just seemed easier since each site couldve potentially had different file uuids.